QuizApp/Client/Account/Models/AccountDetailsNetworkModel.swift

```diff
@@ -0,0 +1,8 @@
+struct AccountDetailsNetworkModel: Codable {
```

Comment:
I prefer to call network models based on the data flow direction:
> body = request (`GetAccountInfoRequestModel`)
> response = response (`AccountInfoResponse`)

Also, use `Encodable` and `Decodable` rather than `Codable`.